packages/features/bookings/lib/getBookingFields.ts

```diff
@@ -338,17 +334,5 @@ export const ensureBookingInputsHaveSystemFields = ({
 
   bookingFields = bookingFields.concat(missingSystemAfterFields);
 
-  bookingFields = bookingFields.map((field) => {
```

Comment:
This was a futuristic code to ensure that system fields can never change their `editable` property because they are essential for the system.

But relying on the name, causes this problem where a custom field with same name as system field when migrates to Bookign Questions, automatically get's marked as a system field because the name is same as System field.

So, remove that futuristic code so that user fields with same name as system fields can be editable.